Error Tests StructuralMechanics

the following test fails on my machine:
~~~
======================================================================
FAIL: test_Uniaxial_KirchhoffSaintVenant_3D (test_constitutive_law.TestConstitutiveLaw)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philipp/software/Kratos_MasterBranch/applications/StructuralMechanicsApplication/tests/test_constitutive_law.py", line 184, in test_Uniaxial_KirchhoffSaintVenant_3D
    self._generic_constitutive_law_test(model_part, deformation_test)
  File "/home/philipp/software/Kratos_MasterBranch/applications/StructuralMechanicsApplication/tests/test_constitutive_law.py", line 176, in _generic_constitutive_law_test
    self.assertAlmostEqual(reference_stress[j], stress[j], 2)
AssertionError: 22265906249999.996 != 22265906250000.004 within 2 places
~~~

the error checking is too strict, I suggest changing the test such that a smaller stress will be the outcome

--- a/applications/StructuralMechanicsApplication/tests/test_constitutive_law.py
+++ b/applications/StructuralMechanicsApplication/tests/test_constitutive_law.py
@@ -172,8 +172,10 @@ class TestConstitutiveLaw(KratosUnittest.TestCase):
 
             stress = cl_params.GetStressVector()
 
+            tolerance = 1.0e-4
             for j in range(cl.GetStrainSize()):
-                self.assertAlmostEqual(reference_stress[j], stress[j], 2)
+                if (abs(stress[j]) > tolerance):
+                    self.assertAlmostEqual((reference_stress[j] - stress[j])/stress[j], 0.0, msg=("Error checking solution " + str(stress[j]) + " different from " + str(reference_stress[j]) + " with tolerance of " + str(tolerance)), delta=tolerance)
 
     def test_Uniaxial_KirchhoffSaintVenant_3D(self):
         # Define a model

